tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -849,6 +849,76 @@ test.describe( 'Nested Tabs tests @nested-tabs', () => {
 
 		await cleanup( wpAdmin );
 	} );
+
+	test( 'Test the nested tabs behaviour when using container flex row @latest', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await setup( wpAdmin );
+		const editor = await wpAdmin.useElementorCleanPost(),
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+
+		await editor.addWidget( 'heading', container );
+
+		const tabsWidgetId = await editor.addWidget( 'nested-tabs', container );
+
+		await editor.addWidget( 'heading', container );
+
+		const contentContainerOne = editor.getPreviewFrame().locator( `.elementor-element-${ tabsWidgetId } .e-n-tabs-content .e-con.e-active` ),
+			contentContainerOneId = await contentContainerOne.getAttribute( 'data-id' ),
+			contentContainerTwo = editor.getPreviewFrame().locator( `.elementor-element-${ tabsWidgetId } .e-n-tabs-content .e-con:not( .e-active )` ).first(),
+			contentContainerTwoId = await contentContainerTwo.getAttribute( 'data-id' ),
+			contentContainerThree = editor.getPreviewFrame().locator( `.elementor-element-${ tabsWidgetId } .e-n-tabs-content .e-con` ).last(),
+			contentContainerThreeId = await contentContainerThree.getAttribute( 'data-id' );
+
+		await editor.getPreviewFrame().waitForSelector( '.e-n-tabs-content .e-con.e-active' );
+
+		// Act.
+		// Add content
+		// Tab 1.
+		await editor.addWidget( 'video', contentContainerOneId );
+
+		// Tab 2.
+		await editor.getPreviewFrame().locator( '.e-normal:not( .e-active )' ).first().click();
+		await editor.addWidget( 'heading', contentContainerTwoId );
+
+		// Tab 3.
+		await editor.getPreviewFrame().locator( '.e-normal:not( .e-active )' ).last().click();
+		await editor.addWidget( 'image', contentContainerThreeId );
+		await editor.addWidget( 'text-editor', contentContainerThreeId );
+
+		// Set container direction to `row`.
+		await editor.selectElement( container );
+		await page.locator( '.elementor-control-flex_direction .eicon-arrow-left' ).click();
+
+		// Assert
+		// Get content container widths.
+		// Tab 1 & Content Container 1.
+		await editor.getPreviewFrame().locator( '.e-normal' ).first().click();
```

Comment:
I suggest that whenever you select `.e-normal` - because it's quite a generic class name - maybe further qualify it by adding the container el that it's in, which is also keeping with the standard that you have used in the rest of this test. We never know in the distant future this class could get into your test in one of the other widgets you've added, and your test will become flaky.
```suggestion
		await editor.getPreviewFrame().locator( '.e-n-tabs-heading .e-normal >> nth=0' ).click();
```

Reply:
Updated